File: app/src/main/java/com/vmenon/mpo/api/ShowDetails.java

package com.vmenon.mpo.api;

import java.util.ArrayList;
import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ShowDetails {

    @SerializedName("name")
    @Expose
    String name;

    @SerializedName("description")
    @Expose
    String description;

    @SerializedName("imageUrl")
    @Expose
    String imageUrl;

    @SerializedName("episodes")
    @Expose
    List<Episode> episodes = new ArrayList<Episode>();

    /**
     * 
     * @return
     *     The name
     */
    public String getName() {
        return name;
    }

    /**
     * 
     * @param name
     *     The name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * 
     * @return
     *     The description
     */
    public String getDescription() {
        return description;
    }

    /**
     * 
     * @param description
     *     The description
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * 
     * @return
     *     The imageUrl
     */
    public String getImageUrl() {
        return imageUrl;
    }

    /**
     * 
     * @param imageUrl
     *     The imageUrl
     */
    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    /**
     * 
     * @return
     *     The episodes
     */
    public List<Episode> getEpisodes() {
        return episodes;
    }

    /**
     * 
     * @param episodes
     *     The episodes
     */
    public void setEpisodes(List<Episode> episodes) {
        this.episodes = episodes;
    }

}

File: app/src/main/java/com/vmenon/mpo/api/ShowDetails.kt
package com.vmenon.mpo.api

import java.util.ArrayList
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ShowDetails(
        /**
         *
         * @return
         * The name
         */
        /**
         *
         * @param name
         * The name
         */
        @SerializedName("name")
        @Expose
        val name: String? = null,

        /**
         *
         * @return
         * The description
         */
        /**
         *
         * @param description
         * The description
         */
        @SerializedName("description")
        @Expose
        val description: String? = null,

        /**
         *
         * @return
         * The imageUrl
         */
        /**
         *
         * @param imageUrl
         * The imageUrl
         */
        @SerializedName("imageUrl")
        @Expose
        val imageUrl: String? = null,

        /**
         *
         * @return
         * The episodes
         */
        /**
         *
         * @param episodes
         * The episodes
         */
        @SerializedName("episodes")
        @Expose
        var episodes: List<Episode> = ArrayList()
)
